Adyen/Model/MarketPay/KYCVerificationResult.cs

```diff
@@ -80,8 +116,12 @@ public override string ToString()
             var sb = new StringBuilder();
             sb.Append("class KYCVerificationResult {\n");
             sb.Append("  AccountHolder: ").Append(AccountHolder).Append("\n");
-            sb.Append("  BankAccounts: ").Append(BankAccounts).Append("\n");
+            sb.Append("  LegalArrangements: ").Append(LegalArrangements).Append("\n");
```

Comment:
These objects are not suitable for printing by default and cause the following errors in LGTM:
`Default 'ToString()': List<KYCLegalArrangementEntityCheckResult> inherits 'ToString()' from 'Object', and so is not suitable for printing.`

Let's see if we do this differently for other object fields. If not; we'll fix it in the model templates later.